Source/Request.swift

```diff
@@ -1315,7 +1315,7 @@ public final class DataStreamRequest: Request {
 
     func didReceive(data: Data) {
         $streamMutableState.write { state in
-            #if !(os(Linux) || os(Windows))
+            #if canImport(Darwin)
```

Comment:
These checks work around limitations in `swift-corelibs-foundation`, so a `Darwin` check isn't appropriate here. In the future these sorts of APIs should be supported across platforms with the `swift-foundation` rewrite.